mmv1/templates/terraform/examples/monitoring_alert_policy_promql_condition.tf.erb

```diff
@@ -0,0 +1,18 @@
+resource "google_monitoring_alert_policy" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
The example must be included in the YAML file like here: https://github.com/jedouard1994/magic-modules/blob/main/mmv1/products/monitoring/AlertPolicy.yaml#L49

Reply:
Done.